File: pinyougou-portal-web/src/main/webapp/js/controller/indexController.js
app.controller('indexController',function ($scope,contentService) {
    //根据分类的ID 查询分类下的广告列表  展示到页面
    $scope.contentList=[];
    $scope.findByCategoryId=function (categoryId) {
        contentService.findByCategoryId(categoryId).success(
            function (response) {//List<content>
                $scope.contentList[categoryId]=response;//List<content>
            }
        )
    }
})

File: pinyougou-portal-web/src/main/java/com/pinyougou/portal/controller/IndexController.java
package com.pinyougou.portal.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.pinyougou.content.service.ContentService;
import com.pinyougou.pojo.TbContent;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

/**
 * @author 三国的包子 ThinkPad pinyougou-parent
 * @version 1.0
 * @description 描述
 * @title 标题
 * @package com.pinyougou.portal.controller
 */
@RestController//@controller +@responsebody
@RequestMapping("/content")
public class IndexController {
    @Reference
    private ContentService contentService;

    @RequestMapping("/findByCategoryId")
    public List<TbContent> findByCategoryId(Long categoryId){
        return contentService.findContentListByCategoryId(categoryId);
    }

    

}
